app/v4/models/response/retrieveLossClaim/RetrieveLossClaimResponse.scala

```diff
@@ -35,7 +35,8 @@ object RetrieveLossClaimResponse extends V4HateoasLinks {
   implicit val writes: OWrites[RetrieveLossClaimResponse] = Json.writes[RetrieveLossClaimResponse]
 
   implicit val reads: Reads[RetrieveLossClaimResponse] = (
-    (JsPath \ "taxYearClaimedFor").read[String].map(TaxYear(_)).map(_.asMtd) and
+    ((JsPath \ "taxYearClaimedFor").read[String].map(TaxYear(_)).map(_.asMtd) orElse
+      (JsPath \ "taxYearClaimedFor").read[Int].map(TaxYear.fromDownstreamInt).map(_.asMtd)) and
```

Comment:
The model tests need updating to reflect this change. Tests are needed for when it is an Int and a string. Please update the reads test in the model spec, thanks